src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -102,7 +126,9 @@ def handle_message(
     message_offset: int,
     message_partition: int,
     topic: str,
+    logical_topic: str,
```

Comment:
Afaict the logical topic is only used to fetch the json codec. Since that's already resolved, can we avoid passing it into handle_message?